packages/cadl-rlc-test/test/openai/cadl-output/src/clientDefinitions.ts

```diff
@@ -1,40 +1,45 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { EmbeddingsParameters, CompletionsParameters } from "./parameters";
 import {
-  Embeddings200Response,
-  EmbeddingsDefaultResponse,
-  Completions200Response,
-  CompletionsDefaultResponse,
+  GetEmbeddingsParameters,
+  GetCompletionsParameters,
+} from "./parameters";
+import {
+  GetEmbeddings200Response,
+  GetEmbeddingsDefaultResponse,
+  GetCompletions200Response,
+  GetCompletionsDefaultResponse,
 } from "./responses";
 import { Client, StreamableMethod } from "@azure-rest/core-client";
 
-export interface Embeddings {
+export interface GetEmbeddings {
   /** Return the embeddings for a given prompt. */
   post(
-    options?: EmbeddingsParameters
-  ): StreamableMethod<Embeddings200Response | EmbeddingsDefaultResponse>;
+    options?: GetEmbeddingsParameters
```

Comment:
why is this parameter called GetEmbeddingsParameters and the method is actually post ?

Reply:
I copied the latest cadl into our repo and I think this is a ask for the Cadl declaration;

https://github.com/Azure/autorest.typescript/blob/2870b491be878b9bdaa2434f4b0d545b62274a83/packages/cadl-rlc-test/test/openai/cadl-output/2022-12-01/openapi.json#L63-L64